build.bat

```diff
@@ -8,5 +8,4 @@ cl.exe /Bv
 cl.exe /nologo /DCOMPILER_MSVC /DNOMINMAX /D_WIN32_WINNT=0x0601 /D_CRT_SECURE_NO_DEPRECATE /D_CRT_SECURE_NO_WARNINGS /bigobj /Zm500 /EHsc /wd4351 /wd4291 /wd4250 /wd4996 /I. /Iinclude /MD /O2 /Oy- /DNDEBUG /wd4117 /Gy /Gw /c slice2.cc
 cl.exe /nologo /DCOMPILER_MSVC /DNOMINMAX /D_WIN32_WINNT=0x0601 /D_CRT_SECURE_NO_DEPRECATE /D_CRT_SECURE_NO_WARNINGS /bigobj /Zm500 /EHsc /wd4351 /wd4291 /wd4250 /wd4996 /I. /Iinclude /MD /O2 /Oy- /DNDEBUG /wd4117 /Gy /Gw /c main.cc
 
-lib.exe /nologo /MACHINE:X64 slice2.obj /ignore:4221
-link.exe /nologo main.obj slice2.lib /SUBSYSTEM:CONSOLE /MACHINE:X64 /DEFAULTLIB:msvcrt.lib /OPT:ICF /OPT:REF
+link.exe /nologo main.obj slice2.obj /SUBSYSTEM:CONSOLE /MACHINE:X64 /DEFAULTLIB:msvcrt.lib /OPT:ICF /OPT:REF
```

Comment:
I don't know Bazel syntax enough to make the same change there.

Reply:
I've made the change.